### PR TITLE
2nd Edition of The Enterprise Path to Service Mesh Architectures

### LINKED ISSUE #883: [Books] Add second edition of The Enterprise Path to Service Mesh Architectures
**Current Behavior**
There is a new edition of this book coming out shortly - https://layer5.io/books/the-enterprise-path-to-service-mesh-architectures

**Desired Behavior**
1. The second edition should be added as a new entry. This description may be used:
```
Planning to build a microservice-driven cloud native application or looking to modernize existing application services? Consider using a service mesh. A service mesh approach can help you create robust and scalable applications, but it also introduces new challenges. This updated report answers common questions regarding service mesh architectures through the lens of a large enterprise. 

Author Lee Calcote, founder and CEO of Layer5, shows developers, operators, architects, and IT leaders how to evaluate your organization’s readiness for using a service mesh—and provides a clear path to help you adopt one. You’ll evaluate several factors when deciding which applications should be built from the ground up and which can be converted with a new service mesh architecture. 

This updated edition discusses several service meshes available and the tools you need to implement them. You’ll explore:

- Service mesh concepts, architecture, and components, such as control planes and data planes
- How a service mesh provides observability, resiliency, security, and traffic control of distributed application services
- Differences among service meshes and service mesh components from several vendors
- How service meshes compare to related technologies, including container orchestrators, API gateways, and client-side libraries
- Practical steps for service mesh adoption, deployment, customization, and integration
- The intelligence of the service mesh management plane and the power of the data plane

Author: Lee Calcote
Publisher: O’Reilly
Publish Date: October 2020
```
2. The first edition entry should remain as-is.

**Book Cover**
Use this image as the book cover for the second edition entry. Resize as needed.
the-enterprise-path-to-service-mesh-architectures_2nd-edition

---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by akashgp09**
**Description**
Added The 2nd Edition of The Enterprise Path to Service Mesh Architectures

This PR fixes #883 

**Screenshots**

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: book
+title:  "The Enterprise Path to Service Mesh Architectures 2nd Edition"
+date:   2018-06-11 21:15:05 +0000
```
Comment (leecalcote) [inline collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md:4]: Let’s bring this date forward to now.

File: collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+image: /assets/images/books/The-Enterprise-path-to-service-mesh-Architectures-2nd-Edition.png
+cover: /assets/images/books/The-Enterprise-path-to-service-mesh-Architectures-2nd-Edition-small.png
+#author: Lee Calcote
+status: published
+brief: "Planning to build a microservice-driven cloud native application or looking to modernize existing application services?"
+description: "A must-read for anyone looking to understand service mesh deployments and get an explanation of service meshes in contrast to related technologies.
+Learn from the largest collection of service mesh projects and their maintainers in the world.Self-study service mesh workshops and tutorials for Istio, Envoy, Linkerd, Consul, App Mesh, Octarine, Kuma, VMware Tanzu Service Mesh"
+redirect_from:
```
Comment (leecalcote) [inline collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md:12]: We can safely remove this redirect. This duplicate will cause conflict Withrow eh existing redirect.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @akashgp09

**Comment by akashgp09** [comment]
Hey @leecalcote are the changes ok ?

**Comment by kishore007k** [comment]
Capture29
The Book Image is not correctly aligned in the mobile view and it would be nice to have the buttons perfectly aligned to the center @akashgp09 Please fix that.

**Comment by akashgp09** [comment]
**We have the alignment issue with all the books images**

image

**Can I open a different issue for it ?**

**Comment by kishore007k** [comment]
> **We have the alignment issue with all the books images**
> 
> image
> 
> **Can I open a different issue for it ?**

@Nikhil-Ladha

**Comment by Nikhil-Ladha** [comment]
I think it will require a common change since all the files use the same code structure. So, I think it can be resolved in this PR itself.

**Comment by akashgp09** [comment]
@Nikhil-Ladha @leecalcote 

image

**Review by kishore007k — DISMISSED** [review]

**Comment by leecalcote** [comment]
Good improvement, @akashgp09

File: collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<div style="margin-left:25px">
+  <li> Service mesh concepts, architecture, and components, such as control planes and data planes </li>
+    <li> How a service mesh provides observability, resiliency, security, and traffic control of distributed application services</li>
+    <li> Differences among service meshes and service mesh components from several vendors</li>
+  <li>How service meshes compare to related technologies, including container orchestrators, API gateways, and client-side libraries</li>
+    <li>Practical steps for service mesh adoption, deployment, customization, and integration</li>
+  <li>The intelligence of the service mesh management plane and the power of the data plane</li>
+
+</div>
+<i>Author: Lee Calcote, Publisher: O'Reilly, Publish Date: October 2020</i>
```
Comment (leecalcote) [inline collections/_books/2020-10-24-the-enterprise-path-to-service-mesh-architectures-2nd-Edition.md:35]:
```suggestion
<i>Author: Lee Calcote, Publisher: O'Reilly, Publish Date: November 2020</i>
```

**Review by leecalcote — APPROVED** [review]
Thank you, @akashgp09. 👍 

Would you like to try adding this to the layer5ng site?

**Comment by akashgp09** [comment]
> Would you like to try adding this to the layer5ng site?

Yes definitely.

**Pull request merged**